feature-pack/src/main/resources/modules/system/layers/base/org/hibernate/5.0/module.xml

```diff
@@ -23,5 +23,28 @@
   -->
 
 <!-- Represents the Hibernate 5.1.x module-->
-<module-alias xmlns="urn:jboss:module:1.5" name="org.hibernate" slot="5.0"  target-name="org.hibernate"/>
+<module xmlns="urn:jboss:module:1.5" name="org.hibernate" slot="5.0">
```

Comment:
shouldn't it be slot="5.1" ?

Reply:
No, it we need to stick with 5.0, since we started with ORM 5.0 initially.